newrelic/hooks/datastore_aioredis.py

```diff
@@ -75,6 +74,9 @@ def _nr_wrapper_AioRedis_method_(wrapped, instance, args, kwargs):
             if isinstance(instance, Pipeline):
                 return wrapped(*args, **kwargs)
 
+        if operation in _redis_client_sync_methods:
+            return wrapped(*args, **kwargs)
+
```

Comment:
With the rearranging of the instrumentation and tests, this should not be needed, but I will test to see if that is the case in a subsequent PR